src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -687,6 +685,58 @@ public AnnotationModel AddAnnotation(string text, Guid id)
         }
 
         /// <summary>
+        /// Sets the event on Annotation model.
+        /// When a node is removed from a group, this event will be
+        /// used to include that node again in that group (UNDO operation)
+        /// </summary>
+        /// <param name="model">The model.</param>
+        internal void SetModelEventOnAnnotation()
```

Comment:
Make this private and invoke it in the constructor.  Also the documentation is out of date.